### PR TITLE
Feature/drag drop

### PR DESCRIPTION
**Pull request by gaber85**

### CODE REVIEW

File: client/src/components/DragDrop/DragDrop.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      this.setState({dragging: true});
+    }
+  };
+
+  handleDragOut = (e) => {
+    e.preventDefault();
+    e.stopPropagation();
+    this.dragCounter -= 1;
+    if (this.dragCounter > 0) return;
+    this.setState({dragging: false});
```
Comment (oliversisson) [inline client/src/components/DragDrop/DragDrop.js:50]: is this the same as
```
    if (this.dragCounter === 0)
        this.setState({dragging: false});
```
?

File: client/src/components/DragDrop/DragDrop.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    return (
+      <div className="drag-drop-container" ref={this.dropRef}>
+      {dragging &&
+        <div className="drag-drop-dash">
+          <div className="drag-drop-inside">
+            <div>Drop Here</div>
+          </div>
+        </div>
+      }
+      Drag and Drop Files
```
Comment (oliversisson) [inline client/src/components/DragDrop/DragDrop.js:77]: nit: indentation

**Review by oliversisson — COMMENTED** [review]
lgtm

**Review by EdwardNeuwirth — APPROVED** [review]
Nice

File: client/src/components/DragDrop/DragDrop.js (hunk 1 above)
Comment (gaber85) [reply client/src/components/DragDrop/DragDrop.js:50]: I'll keep it like this for now, and then try without once it's up and running.

**Pull request merged**